h2o-docs/src/booklets/v2_2015/source/GLMBooklet.tex

```diff
@@ -994,8 +994,7 @@ \subsubsection{P-Values}
 (insiginificant) while a low p-value suggest that the coefficient is statistically significant.
 
 You can request p-values by setting the \\\texttt{compute\_p\_values} option. It can only be used with the
-\texttt{IRLSM} solver and no regularization. It is recommended that you also set the
-\texttt{remove\_collinear\_columns} option. Otherwise, H2O will return an error if it detects collinearity in the dataset and p-values are requested. 
+\texttt{IRLSM} solver. P-values and z-score are compatible with regularization. It is recommended that you also set the \texttt{remove\_collinear\_columns} option. Otherwise, H2O will return an error if it detects collinearity in the dataset and p-values are requested. 
```

Comment:
Hannah:  can you change

P-values and z-score are compatible with regularization

to

P-values and z-score are computed with or without regularization

please.  Not sure what compatible means.